src/implementations/vanilla/components/footer-standardised/footer-standardised-ec.scss

```diff
@@ -155,96 +168,7 @@ $_column-gap: map.get(theme.$grid-gutter-width, 'm');
     margin-top: map.get(theme.$spacing, 'xl');
   }
 
-  .ecl-footer-standardised__list--columns {
-    column-count: 2;
-    column-gap: $_column-gap;
-  }
-
   .ecl-footer-standardised__link {
```

Comment:
I guess that the `.ecl-footer-standardised__list--columns` class (or something equivalent) will still be needed for the core footer, but we'll see then.